test/es2015/module-support-assessment-test.mjs

```diff
@@ -0,0 +1,14 @@
+"use strict";
+import referee from "referee";
+import sinon from "../../lib/sinon";
+import * as aModule from "./a-module";
+const {assert} = referee;
+
+describe("Explicit lack of support for stubbing Modules", function() {
+    it("should give a proper error message for modules without default export", function() {
```

Comment:
Any export (or rather import) would be read-only, wouldn't it? This seems to suggest that read-only only applies to `default` exports.

Reply:
I was thinking of adding some additional tests, but couldn't be bothered at that time of the night :-) I added some more tests now, and as you can see, it's only-halfway true. You can actually stub some forms of exports. Although you can't modify the exported properties, one can modify the objects that has been exported (check the content of the tests), so one can modify the props of `export default { a: 1, b: 2 }`, just not point `default` to another object.